core/loadpoint.go

```diff
@@ -652,9 +656,12 @@ func (lp *Loadpoint) setLimit(chargeCurrent float64, force bool) error {
 	// set enabled/disabled
 	if enabled := chargeCurrent >= lp.GetMinCurrent(); enabled != lp.enabled {
 		if remaining := (lp.GuardDuration - lp.clock.Since(lp.guardUpdated)).Truncate(time.Second); remaining > 0 && !force {
-			lp.log.DEBUG.Printf("charger %s: contactor delay %v", status[enabled], remaining)
+			lp.publishTimer(guardTimer, lp.GuardDuration, guardEnable)
 			return nil
 		}
+		if lp.guardUpdated != elapsed {
```

Comment:
was tut das?

Reply:
Stellt sicher, dass wir `.elapseGuard()` nicht permanent aufrufen sondern nur wenn da auch wirklich ein Timer ist. Alternative Vorschläge? Wir können das auch in die `elapseGuard` Funktion verschieben, weiß aber nicht ob das schicker ist.

Reply:
Aber warum brauchen wir das? Hat doch bisher auch ohne funktioniert?

Reply:
Vorher gab es keinen gepublishten Timer der zurückgesetzt werden musste.

Reply:
Ahh. Dann lass uns doch das if in die elapse Funktion packen, find ich etwas klarer (= an der Stelle wird immer "elapsed").

Reply:
done